apps/dash-study-browser/app.py

```diff
@@ -0,0 +1,326 @@
+app.layout = html.Div(
+    children=[
+        # Error Message
+        html.Div(id="error-message"),
+        # Top Banner
+        html.Div(
+            className="study-browser-banner row",
+            children=[
+                html.H2(className="h2-title", children="ANIMAL STUDY BROWSER"),
+                html.Div(
+                    className="div-logo",
+                    children=html.Img(
+                        className="logo", src=app.get_asset_url("dash-logo-new.png")
+                    ),
+                ),
+                html.H2(className="h2-title-mobile", children="ANIMAL STUDY BROWSER"),
+            ],
+        ),
+        # Body of the App
+        html.Div(
+            className="row app-body",
+            children=[
+                # User Controls
+                html.Div(
+                    className="four columns card",
+                    children=[
+                        html.Div(
+                            className="bg-white user-control",
+                            children=[
+                                html.Div(
+                                    className="padding-top-bot",
+                                    children=[
+                                        html.H6("Test Articles"),
+                                        dcc.Dropdown(id="study-dropdown"),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="padding-top-bot",
+                                    children=[
+                                        html.H6("Choose the type of plot"),
+                                        dcc.RadioItems(
+                                            id="chart-type",
+                                            options=[
+                                                {"label": "Box Plot", "value": "box"},
+                                                {
+                                                    "label": "Violin Plot",
+                                                    "value": "violin",
+                                                },
+                                            ],
+                                            value="violin",
+                                            labelStyle={
+                                                "display": "inline-block",
+                                                "padding": "12px 12px 12px 0px",
+                                            },
+                                        ),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="padding-top-bot",
+                                    children=[
+                                        html.H6("CSV File"),
+                                        dcc.Upload(
+                                            id="upload-data",
```

Comment:
You can specify filetypes that are accepted (still a good idea to keep that check in the callback, but this way you will avoid issues that stem from non-CSV files being uploaded). https://dash.plot.ly/dash-core-components/upload (`accept` argument)